.github/actions/kibana-ftr/action.yml

```diff
@@ -0,0 +1,79 @@
+name: 'Run Kibana FTR'
+description: 'Run Kibana tests using FTR'
+
+inputs:
+  test_kibana_url:
+    description: 'URL for the Kibana instance to test'
+    required: true
+  test_es_url:
+    description: 'URL for the ES instance'
+    required: true
+  es_version:
+    description: 'Version of Elasticsearch to test against'
+    required: true
+  ref_commit_sha:
+    description: 'Kibana PR commit sha'
+    required: false
+    default: 'main'
+  test_browser_headless:
```

Comment:
isn't it headless by default, since it's running from CI? not sure how we'd access a browser here

Reply:
You're correct. I'll set `TEST_BROWSER_HEADLESS: '1'` by default and remove the input parameters. Since we're running from CI, accessing a browser directly isn't feasible.